types/cleave.js/cleave.js-tests.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from "react";
+import CleaveReact, { CleaveProps } from "cleave.js/react";
```

Comment:
Since this is a callable or constructable entity, it isn't legal in ES2015 to import this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```